### PR TITLE
[2.4] Added rancher-externalip-webhook embargo cve chart

### PR DESCRIPTION
**Pull request by rawmind0**

### CODE REVIEW

File: charts/rancher-externalip-webhook/v0.0.1-rc2/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- externalip
+- webhook
+- security
+maintainers:
+- email: raul@rancher.com
+  name: Raul Sanchez
+name: rancher-externalip-webhook
+sources:
+- https://github.com/rancher/externalip-webhook
+version: 0.0.1-rc2
```
Comment (cbron) [inline charts/rancher-externalip-webhook/v0.0.1-rc2/Chart.yaml]: Lets start at 0.1.1

File: charts/rancher-externalip-webhook/v0.0.1-rc2/values.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  caBundle: ""
+  # If cert-manager integration is disabled, upload certs data (ca.crt, tls.crt and tls.key) as k8s secretName in the namespace
+  secretName: webhook-server-cert
+## Details about the image to be pulled.
+image:
+  pullPolicy: IfNotPresent
+  pullSecrets: []
+  # community image
+  #repository: k8s.gcr.io/multitenancy/externalip-webhook
+  #tag: v1.0.0
```
Comment (cbron) [inline charts/rancher-externalip-webhook/v0.0.1-rc2/values.yaml]: Needs this to be available

File: charts/rancher-externalip-webhook/v0.0.1-rc2/values.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  # Enable webhook metrics export to Prometheus
+  prometheusExport: false
+  # Webhook metrics auth proxy
+  authProxy:
+    enabled: false
+    port: 8080
+    image:
+      pullPolicy: IfNotPresent
+      pullSecrets: []
+      repository: gcr.io/kubebuilder/kube-rbac-proxy
```
Comment (cbron) [inline charts/rancher-externalip-webhook/v0.0.1-rc2/values.yaml]: This needs to be mirrored to Rancher. All repository images must start with rancher/.

File: charts/rancher-externalip-webhook/v0.0.1-rc2/values.yaml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  requests:
+    memory: 20Mi
+    cpu: 100m
+service:
+  metricsPort: 8443
+  webhookPort: 443
+## Webhook serviceAccountName. Just used if metrics.authProxy.enabled = false
+serviceAccountName: default
+## List of node taints to tolerate (requires Kubernetes >= 1.6)
+tolerations: []
```
Comment (cbron) [inline charts/rancher-externalip-webhook/v0.0.1-rc2/values.yaml]: Need to add linux taints/tolerations as outlined in doc

File: charts/rancher-externalip-webhook/v0.0.1-rc2/values.yaml (hunk 2 above)
Comment (rawmind0) [reply charts/rancher-externalip-webhook/v0.0.1-rc2/values.yaml]: Removed at https://github.com/rancher/externalip-webhook/pull/3

File: charts/rancher-externalip-webhook/v0.0.1-rc2/values.yaml (hunk 3 above)
Comment (rawmind0) [reply charts/rancher-externalip-webhook/v0.0.1-rc2/values.yaml]: Requested https://github.com/rancher/image-mirror/pull/58

File: charts/rancher-externalip-webhook/v0.0.1-rc2/values.yaml (hunk 4 above)
Comment (rawmind0) [reply charts/rancher-externalip-webhook/v0.0.1-rc2/values.yaml]: Added at https://github.com/rancher/externalip-webhook/pull/3

File: charts/rancher-externalip-webhook/v0.0.1-rc2/Chart.yaml (hunk 1 above)
Comment (rawmind0) [reply charts/rancher-externalip-webhook/v0.0.1-rc2/Chart.yaml]: updated

**Review by cbron — APPROVED** [review]
Lint looks unrelated

**Pull request merged**